internal/uvm/network.go

```diff
@@ -642,3 +642,13 @@ func (uvm *UtilityVM) RemoveAllNICs(ctx context.Context) error {
 	}
 	return nil
 }
+
+// UpdateNIC updates a UVM's network adapter.
+func (uvm *UtilityVM) UpdateNIC(ctx context.Context, id string, settings interface{}) error {
```

Comment:
Any merit in just having this take in hcsshema.NetworkAdapter instead of interface? If we stay with interface, can we try and type cast and see if we're A-OK before making the req?

Reply:
Good point, it's never called with anything else anyways.